### ISSUE TITLE
Events should have an accessibility field specifying location-specific issues.

### ISSUE THREAD

**Issue by harryslimes** [project-accelerate/m.app#28]
Possible:
Should include boilerplate text with email and phone number of organisers contact details to help with accessibility
